lua/starfall/libs_sh/trace.lua

```diff
@@ -66,16 +74,21 @@ end
 -- @param boolean? ignworld Whether the trace should ignore world
 -- @return table Result of the trace https://wiki.facepunch.com/gmod/Structures/TraceResult
 function trace_library.line(start, endpos, filter, mask, colgroup, ignworld)
+	vec_SetUnpacked(start_vec, start[1], start[2], start[3])
+	vec_SetUnpacked(endpos_vec, endpos[1], endpos[2], endpos[3])
+
 	return structWrapper(instance, util_TraceLine({
-		start = vunwrap(start),
-		endpos = vunwrap(endpos),
+		start = start_vec,
+		endpos = endpos_vec,
 		filter = convertFilter(filter),
 		mask = mask,
 		collisiongroup = colgroup,
 		ignoreworld = ignworld,
 	}), "TraceResult")
 end
 
+local minbox_vec, maxbox_vec = Vector(0, 0, 0), Vector(0, 0, 0)
```

Comment:
Can you put all the locals together on one line?

Reply:
done